app/utils/clone-learner-group.js

```diff
@@ -0,0 +1,32 @@
+import RSVP from 'rsvp';
```

Comment:
just curious why this imports from `rsvp` directly, and not via `Ember.RSVP`. 
should we do this elsewhere too?

Reply:
This change will actually happen for us automatically with a code mode I have in #3120 so we don't need to worry about it too much. It is the new right way to do this, but in most cases we need the Ember object anyway so until that goes away either way is fine.